src/qibo/optimizers/gradient_based.py

```diff
@@ -1 +1,83 @@
 """Gradient descent strategies to optimize quantum models."""
+
+import tensorflow as tf
+
+from qibo.backends import TensorflowBackend
+from qibo.config import log, raise_error
+from qibo.optimizers.abstract import Optimizer, check_options
+
+
+class TensorFlowSGD(Optimizer):
```

Comment:
Stupid remark: use at least a consistent case for `Tensorflow` (I know the official one is `TensorFlow`, but if there is already one around, better to sync the two of them)